force-app/main/default/classes/CountTasksTest.cls

```diff
@@ -0,0 +1,206 @@
+/**
+ * Apex Test Class for all scenarios
+ * (insert, update, delete, close tasks,
+ * re open tasks, reparenting tasks)
+ */
+@isTest
+public class CountTasksTest {
+  @TestSetup
+  private static void makeData() {
+    List<Account> accounts = new List<Account>();
+
+    for (Integer i = 0; i < 100; i++) {
+      Account acc = new Account(
+        Name = 'Test Account ' + i,
+        Type = 'Prospect',
+        RecordTypeId = TaskTriggerHandler.recordTypeUS
+      );
+      accounts.add(acc);
+    }
+
+    insert accounts;
+  }
+
+  //inserting >200 tasks
+  @isTest
+  public static void insertNewTasks() {
```

Comment:
please read up on test.starttest() and test.stoptest() and apply it to your (and future) test class(es)